src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -329,7 +329,7 @@ $.BootstrapTable = class extends $.BootstrapTable {
   onSearch (event) {
     super.onSearch(event)
 
-    if ($(event.currentTarget).parent().hasClass('search')) {
+    if (this.options.search) {
```

Comment:
Is this change equivalent?

Reply:
Fix `$(event.currentTarget).parent().hasClass('search')` always return `false` bug.